generic/tests/cfg/iozone_windows.cfg

```diff
@@ -3,10 +3,21 @@
     only Windows
     type = iozone_windows
     cdrom_cd1 = isos/windows/winutils.iso
+    i386:
+        devcon_dirname = "x86"
+    x86_64:
+        devcon_dirname = "amd64"
+    devcon_path = "WIN_UTILS:\devcon\${devcon_dirname}\devcon.exe"
+    driver_enable_cmd = 'WIN_UTILS:\devcon\${devcon_dirname}\devcon.exe enable "@DEVICE_ID"'
```

Comment:
Maybe we can use ${devcon_path} to replace WIN_UTILS:\devcon\${devcon_dirname}\devcon.exe?